.github/workflows/vulnerability-scanner-generate-database.yml

```diff
@@ -38,7 +38,6 @@ jobs:
         with:
           submodules: recursive
           fetch-depth: 0
-          ref: ${{ matrix.wazuh_version }}
 
       # Checkout to the tag. If it doesn't exist, continue with the branch
       - name: Checkout to tag
```

Comment:
We can rename this and the "Checkout to branch" steps to better reflect the intent

Reply:
done